judo-runtime-core-jsl-itest/pom.xml

```diff
@@ -66,6 +66,7 @@
                                 <source>${project.basedir}/target/generated-sources/model/sdk/Abstract</source>
                                 <source>${project.basedir}/target/generated-sources/model/sdk/TernaryTest</source>
                                 <source>${project.basedir}/target/generated-sources/model/sdk/ContainsModel</source>
+                                <source>${project.basedir}/target/generated-sources/model/sdk/DefaultRequiredValue</source>
```

Comment:
Primitive related tests should go to the `Primitives.jsl`. Creating dedicated models for every use-case slows down the generation/test process. We already have similar tests there anyway.